modules/bgsegm/include/opencv2/bgsegm.hpp

```diff
@@ -183,10 +183,50 @@ class CV_EXPORTS_W BackgroundSubtractorGMG : public BackgroundSubtractor
 @param decisionThreshold Threshold value, above which it is marked foreground, else background.
  */
 CV_EXPORTS_W Ptr<BackgroundSubtractorGMG> createBackgroundSubtractorGMG(int initializationFrames=120,
-                                                                        double decisionThreshold=0.8);                                  
+                                                                        double decisionThreshold=0.8);
 
 //! @}
 
+/** @brief Background subtraction based on counting.
```

Comment:
Please put code below before this line above:
```
//! @}
```
It is required for proper documentation generation.